store/activity.go

```diff
@@ -352,8 +352,12 @@ func findActivityImpl(ctx context.Context, tx *Tx, find *api.ActivityFind) ([]*a
 		}
 		where = append(where, fmt.Sprintf("(%s)", strings.Join(queryValues, " OR ")))
 	}
-	if v := find.Level; v != nil {
-		where, args = append(where, fmt.Sprintf("level = $%d", len(args)+1)), append(args, *v)
+	if v := find.LevelList; v != nil {
+		var queryValues []string
+		for _, level := range v {
+			queryValues, args = append(queryValues, fmt.Sprintf("level = $%d", len(args)+1)), append(args, level)
+		}
+		where = append(where, fmt.Sprintf("(%s)", strings.Join(queryValues, " OR ")))
```

Comment:
We need (x or x or x)

Reply:
For now, our query statement will look like `"\n\t\tSELECT\n\t\t\tid,\n\t\t\tcreator_id,\n\t\t\tcreated_ts,\n\t\t\tupdater_id,\n\t\t\tupdated_ts,\n\t\t\tcontainer_id,\n\t\t\ttype,\n\t\t\tlevel,\n\t\t\tcomment,\n\t\t\tpayload\n\t\tFROM activity\n\t\tWHERE 1 = 1 AND (level = $1 OR level = $2)"`.

It's already (level = $1 OR level = $2). Do you mean to use lower cases?